Libplanet.Tests/Action/ActionEvaluatorTest.cs

```diff
@@ -1100,6 +1100,85 @@ private async Task CheckGenesisHashInAction()
             Assert.Equal(chain.Genesis.Hash, evaluations[0].InputContext.GenesisHash);
         }
 
+        [Fact]
+        private void GenerateRandomSeed()
+        {
+            byte[] preEvaluationHashBytes =
+            {
+                0x45, 0xa2, 0x21, 0x87, 0xe2, 0xd8, 0x85, 0x0b, 0xb3, 0x57,
+                0x88, 0x69, 0x58, 0xbc, 0x3e, 0x85, 0x60, 0x92, 0x9c, 0xcc,
+            };
+            var signature = new byte[]
```

Comment:
```suggestion
            byte[] signature =
```
Just for consistency with above😅